neo4j-gremlin/pom.xml

```diff
@@ -50,6 +50,11 @@ limitations under the License.
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.tinkerpop</groupId>
+            <artifactId>tinkergraph-gremlin</artifactId>
+            <version>${project.version}</version>
```

Comment:
Nit: Can this be limited to `<scope>test</scope>`?

Reply:
oops - that's more than just a nit....thanks for noticing that.